types/backbone/index.d.ts

```diff
@@ -245,7 +245,7 @@ declare namespace Backbone {
          */
         url: () => string;
 
-        urlRoot: any;
+        urlRoot: _Result<string>;
```

Comment:
I'm a little surprised the types of `url` and `urlRoot` don't match.  This one can return a string directly?

Reply:
Yep, example from docs https://backbonejs.org/#Model-urlRoot or from sources, it call `_.result` https://github.com/jashkenas/backbone/blob/master/backbone.js#L703